lib/matplotlib/tests/test_backend_inline.py

```diff
@@ -13,7 +13,7 @@
 
 
 def test_ipynb():
-    nb_path = Path(__file__).parent / 'test_inline_01.ipynb'
+    nb_path = Path(__file__).parent / 'data/test_inline_01.ipynb'
```

Comment:
```suggestion
    nb_path = Path(__file__).parent / 'data' / 'test_inline_01.ipynb'
```
Is it a bit safer to construct the paths like this, for platforms (Windows) that don't use '/' as the path delimeter?

Reply:
`Path` normalizes separators; there is no difference between the two options:
```
>>> from pathlib import Path
>>> Path('foo') / 'bar/baz'
WindowsPath('foo/bar/baz')
>>> str(Path('foo') / 'bar/baz')
'foo\\bar\\baz'

>>> Path('foo') / 'bar' / 'baz'
WindowsPath('foo/bar/baz')
>>> str(Path('foo') / 'bar' / 'baz')
'foo\\bar\\baz'
```
(Also Windows supports both anyway.)